javascript-timers/interval.js

```diff
@@ -0,0 +1,23 @@
+var $heading = document.querySelector('h1');
+function changeNumber() {
+  if ($heading.textContent === '4') {
+    $heading.textContent = '3';
+  } else if ($heading.textContent === '3') {
+    $heading.textContent = '2';
+  } else if ($heading.textContent === '2') {
+    $heading.textContent = '1';
+  } else {
+    $heading.textContent = '~Earth Beeeelooowww Us~'
+    clearCountdown();
+  }
```

Comment:
Try moving your countdown number in to a variable instead of checking the DOM.

```js
var count = 4;
```

That should help cut down on unnecessary conditional logic.